tests/st/calicoctl/test_crud.py

```diff
@@ -486,8 +492,9 @@ def test_bgpconfig(self):
         rc.assert_error(SET_DEFAULT)
 
         # Delete the resource by name (i.e. without using a resource version).
+        # Note that deleting the default bgpconfig resource is disallowed.
         rc = calicoctl("delete bgpconfig %s" % name(rev3))
-        rc.assert_no_error()
+        #rc.assert_no_error()
```

Comment:
Is there an "assert_error" method we can call?

Reply:
actually this works. I think my error and fix were caused by re-running with the same `etcd` server instance.